.github/workflows/pull-requests-release.yaml

```
@@ -39,58 +39,100 @@ jobs:
+
+      - name: Create tag on merge commit
         run: |
-          git tag ${{ steps.get_tag.outputs.tag }} ${{ github.sha }} --force
-          git push origin ${{ steps.get_tag.outputs.tag }} --force
-  
+          git tag -f ${{ steps.get_tag.outputs.tag }} ${{ github.sha }}
+          git push -f origin ${{ steps.get_tag.outputs.tag }}
+
+      # Get latest published release
+      - name: Get latest published release
+        if: steps.check_release.outputs.skip == 'false'
```

actionlint (1.7.4)

property "check_release" is not defined in object type {get_tag: {conclusion: string; outcome: string; outputs: object}}

(expression)